Sources/swift-parser-cli/swift-parser-cli.swift

```diff
@@ -198,17 +198,22 @@ class PrintDiags: ParsableCommand {
   @Flag(name: .long, help: "Perform sequence folding with the standard operators")
   var foldSequences: Bool = false
 
-  @Flag(name: .long, help: "Colorize output with ANSI color codes")
+  @Flag(name: .long, help: "Force output coloring with ANSI color codes")
```

Comment:
You could make this a flag with `inversion: .prefixedNo` like in https://github.com/apple/swift-format/blob/072768f1af50989996e137919f9f6be71e8294e6/Sources/swift-format/Subcommands/LintFormatOptions.swift#L62-L69 so you can also force coloring off if you want to.

---

And now that I think about it, we should name the flag `color-diagnostics` (just rename the variable to `colorDiagnostics`) to match the name of the flag in the Swift Compiler.

```suggestion
  @Flag(name: .long, inversion: .prefixedLo, help: "Enable or disable coloring with ANSI color codes")
  let colorDiagnostics: Bool?
```